### ISSUE TITLE
Upgrade from 7.x to 8.x causes random corruption in enviroment.

### ISSUE THREAD

**Issue by ghost** [a85/POSTMan-Chrome-Extension#9829]
**Describe the Issue**
I upgraded from 7.x to 8.1. After upgrade, my Postman environment stopped working. I attempted to upgrade a couple of times, and it broke both times, but differently. It seems that it actually broke my workspace, as when I re-installed 7.x it no longer worked.
The first time it broke, I was unable to figure out exactly what had broken, however, I am using a bearer token, and auto populating the token from a script running from my Auth API in the Tests. Auth worked perfectly, the bearer token seemed to be set correctly, however, I could not authenticate subsequent API calls. Taking the "code" produced (in Python) and pasting it into a script, keeping the bearer token in tact, and the script would authenticate and execute correctly.
I re-installed 7.x and my workspace was still broken in the same way when it connected to my saved workspace, so it seems that something in my workspace is messed up.
The second time, I re-created a workspace in 7.x and set up up the same way. I upgraded to 8.1 again, and once again, it did not work after the upgrade. However, it broke in a different way. This time, a URL that I had saved into an environmental variable had a %0A added into it, and hence my API call failed.

To summarize, it seems that something is getting corrupt in my workspace in the upgrade from 7.x to 8.1, but it does not seem to be the same thing each time.
My workspace has a single collection in it that was imported from a YAML file with ~160 URLs in it, and also has aound 9 different environments each set up as follows:
baseUrl {{hostname}}/api/v1.0/gateway
authUrl {{hostname}}/api/v1.0/auth
hostname https://<myhostname>
authToken <autoSet>

**Steps to Reproduce**
Use a workspace in 7.x and upgrade to 8.1.

**Environment Information**

 - Platform Type: MacBook Pro (16-inch, 2019)
 - Postman Version: From 7.x to 8.1.0
 - Operating System: OSX 10.15.7 

**Additional Context**

**Comment by shubhbhargav** [comment]
@colross Are you still facing this issue?
